src/test/mod.rs

```diff
@@ -426,6 +431,16 @@ fn check_files(files: Vec<PathBuf>, opt_config: &Option<PathBuf>) -> (Vec<Format
     let mut reports = vec![];
 
     for file_name in files {
+        let sig_comments = read_significant_comments(&file_name);
```

Comment:
It would be better if the changes on this file were done in a separate PR.

Reply:
See #3558